app/javascript/styles/_explore.scss

```diff
@@ -209,3 +209,12 @@
 	}
 }
 
+.de-download-button{
+	margin-right: 0.5em;
+	margin-left: 1em;
+	display: inline-block;
+	vertical-align: middle;
+	padding: 2px 5px;
+	border-radius: 4px;
+  }
+  
```

Comment:
Could you refine this and the `.de-modal-raw-wrapper` CSS rule to use SCP-standard 2-space indentation?

Reply:
I can change those. I based the spacing for`.de-download-button` off of `.de-modal-row-wrapper`, do we have a linter or documented somewhere for the SCP-standard for indentation?

Reply:
Actually I'm seeing a bit of differing indentation in the css file, I think if we want to standardize on 2-spaces that should be it's own work.
![Screen Shot 2023-01-12 at 1 34 10 PM](https://user-images.githubusercontent.com/54322292/212150383-86165af2-42a6-4ed6-853c-362cd974a789.png)

Reply:
Makes sense!  So for future individual cases, let's use 2 spaces.  I agree that implementing that for all our SCSS is worth a separate task.